app/lib/rocketchat/rocketchat.js

```diff
@@ -703,6 +703,10 @@ const RocketChat = {
 
 		data = data.slice(0, 7);
 
+		if (searchView) {
+			return data;
+		}
+
 		data = data.map(sub => ({
```

Comment:
We probably want to add extra info here.

Reply:
Extra info as in? I mean the data required to display a room card is return here white the data object returned below lack some of the fields. Do we need some sort of comment or something here?

Reply:
Or should we return the existing data object with the extra field in all cases regardless of where is the function called?

Reply:
It would work just fine if we return the whole object without the searchView as well. I just thought that there is some reason behind returning some fields only and not the whole object (Maybe some performance benefit or something) .